package.json

```
@@ -3,7 +3,7 @@
   "private": true,
   "dependencies": {
     "@rails/actioncable": "^6.0.0",
-    "@rails/ujs": "^6.0.0",
+    "@rails/ujs": "7.0.0",
```

Can we use `^7.0.0`?

```suggestion
    "@rails/ujs": "^7.0.0",
```

If this is a temporary update (we remove this after we complete this Rails upgrade), I'm OK with `7.0.0`.

---

Yes. I want to add this changes as a temporary because I want to use the same version as Rails version 7.